ui/src/app/state/user/user.reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { Action, createReducer, on } from "@ngrx/store";
+import { User } from "../../services";
+import { setUser } from "./user.actions";
+
+export interface UserState {
```

Comment:
we might do
```suggestion
export interface UserState extends User { }
```
or do we expect mre props on the user state?
else we can name it "loggedInUser" then its more clear what it is. and that naming there would not be so "weird" with "user: user:...